cdc/sink/producer/kafka/kafka.go

```diff
@@ -285,7 +288,13 @@ func (k *kafkaSaramaProducer) run(ctx context.Context) error {
 		case <-k.closeCh:
 			return nil
 		case <-ticker.C:
-			k.metricsMonitor.CollectMetrics()
+			// if error returned, it means the kafka broker clusters is unreachable,
+			// it's ok to return error. Most of the time, this only happen when no
+			// data sending, since if there is any data sending to the kafka,
+			// `ErrKafkaAsyncSendMessage` would have a higher chance to be triggered.
+			if err := k.metricsMonitor.CollectMetrics(); err != nil {
```

Comment:
metrics collect failed should not stop the changefeed

Reply:
yes, I will revise it twice.